### ISSUE TITLE
ROL errors in blake build

### ISSUE THREAD

**Issue by ikalash** [sandialabs/Albany#796]
It looks like there are ROL errors (not just in the tests) in the nightly blake builds: 

https://sems-cdash-son.sandia.gov/cdash/viewBuildError.php?buildid=29826

I believe this is with the deprecated Tpetra code turned off.

**Comment by jewatkins** [comment]
It looks like it is just from building tests? Although the error would probably come in anyways through the Albany build.

**Comment by jewatkins** [comment]
Should I try turning the tests off?

**Comment by mperego** [comment]
This is not new right? I think we discussed this before and we decided to wait until ROL fixes this (they are aware of it). Alternatively, we can use the default settings for the tpetra deprecated codes.

**Comment by jewatkins** [comment]
Ah okay, I see what happened. So the configure script on blake had deprecated code on:
https://sems-cdash-son.sandia.gov/cdash/build/29495
but the script in the Albany repo was set to off (which I then turned on then back off):
https://github.com/sandialabs/Albany/commit/5c2ad77b5dc9603249e8626e8cfac29bd2c2af9c
https://github.com/sandialabs/Albany/commit/77983a7fdfcd5318e4e67f39fa5b393abb53f470

I probably forgot or wasn't aware of the discussion to turn it on. I can turn tpetra deprecated code back on for this build only?

**Comment by mperego** [comment]
Yeah.. or simply remove the cmake option and use the defaults

**Comment by jewatkins** [comment]
Should we start using the defaults for everything? Or just this build?

**Comment by bartgol** [comment]
So these errors are still related to ROL unit tests, right? Can't we disable all tests in Trilinos? What's the point of building them?

**Comment by ikalash** [comment]
You're right, but I thought we had turned off the tests.  @jewatkins would you be able to do this?

**Comment by jewatkins** [comment]
Yeah I can try turning the tests off first and check back tomorrow.

**Comment by jewatkins** [comment]
I forgot to check this Friday but this was fixed based on the dashboard results from that day.
https://sems-cdash-son.sandia.gov/cdash/index.php?project=Albany&date=2022-02-25
